common/src/main/proto/TransportMessages.proto

```diff
@@ -402,10 +404,26 @@ message PbUnregisterShuffle {
   string requestId = 3;
 }
 
+message PbBatchUnregisterShuffles {
+  string appId = 1;
+  string requestId = 2;
+  repeated int32 shuffleIds = 3;
+}
+
 message PbUnregisterShuffleResponse {
   int32 status = 1;
 }
 
+message PbBatchUnregisterShuffleResponses {
+  int32 status = 1;
+  repeated PbUnregisterShuffleResponsesInfo unregisterShuffleResponsesInfo = 2;
+}
+
+message PbUnregisterShuffleResponsesInfo {
```

Comment:
I think this proto is unnecessary. Maybe 
```
message PbBatchUnregisterShuffleResponses {
  int32 status = 1;
  repeated int32 shuffleIds = 2;
}
```

Reply:
According to your implementation, the shuffleStatus can be ignored because there is no logic to process the failure of errors that might occur when removing the shuffle ID.

Reply:
If a batch unregistered shuffle RPC fails, retrying it can be enough.